doc/developers/contributing.rst

```diff
@@ -872,6 +872,51 @@ from high-level questions to a more detailed check-list.
   instructive?
 
 
+Standard replies
+................
+
+It may be helpful to store some of these in GitHub's [saved
+replies](https://github.com/settings/replies/) for reviewing:
+
+Issue: Update the docs svp
+    ::
+        Please feel free to offer a pull request updating the documentation if
+        you feel it could be improved.
+
+PR-NEW: Better description
+    ::
+        Please make the title of the PR descriptive so that we can easily
+        recall the issue it is resolving.  You should state what issue it
```

Comment:
Will it be better to state "what issues/PRs it resolves"? It's more consistent with the contribution guide. Also, I believe it is useful to mention issues and PRs here.